server/src/main/java/com/broadcom/lsp/cobol/service/delegates/validations/AnalysisResult.java

```diff
@@ -30,23 +29,25 @@
  * Diagnostic} grouped by related documents, all the
  */
 @Value
-@AllArgsConstructor
 @Builder(toBuilder = true)
 public class AnalysisResult {
-  private Map<String, List<Diagnostic>> diagnostics;
-  private Map<String, List<Location>> variableDefinitions;
-  private Map<String, List<Location>> variableUsages;
-  private Map<String, List<Location>> paragraphDefinitions;
-  private Map<String, List<Location>> paragraphUsages;
-  private Map<String, List<Location>> constantDefinitions;
-  private Map<String, List<Location>> constantUsages;
-  private Map<String, List<Location>> copybookDefinitions;
-  private Map<String, List<Location>> copybookUsages;
-  private List<DocumentSymbol> outlineTree;
+  Map<String, List<Diagnostic>> diagnostics;
+  Map<String, List<Location>> variableDefinitions;
+  Map<String, List<Location>> variableUsages;
+  Map<String, List<Location>> paragraphDefinitions;
+  Map<String, List<Location>> paragraphUsages;
+  Map<String, List<Location>> sectionDefinitions;
+  Map<String, List<Location>> sectionUsages;
+  Map<String, List<Location>> constantDefinitions;
+  Map<String, List<Location>> constantUsages;
+  Map<String, List<Location>> copybookDefinitions;
+  Map<String, List<Location>> copybookUsages;
+  List<DocumentSymbol> outlineTree;
 
   public static AnalysisResult empty() {
     return new AnalysisResult(
-        Map.of(), Map.of(), Map.of(), Map.of(), Map.of(), Map.of(), Map.of(), Map.of(), Map.of(),
+        Map.of(), Map.of(), Map.of(), Map.of(), Map.of(), Map.of(),  Map.of(), Map.of(),
```

Comment:
I suggest to introduce constructor (chain) to which provides all theses init-values by default. Otherwise it will be reteated all the like this way.